MovieTime/MovieTime/Core/MTIdentifiers.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MTIdentifiers.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 10/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+struct MTIdentifiers {
+
+    struct Storyboard {
+        struct Main {
+            static let name = "Main"
```

Comment:
why do you prefer to store strings in structs rather than enums?

Reply:
This is the place with important project constants: Storyboard / Persistency Keys / etc. So, I decided to because in the future (or project growth) it will be possible to have different types of constants of certain subject.
And enums do not support this (Strings and Integers in the same enum):
Example:

```
struct Storyboard {
    struct Main {
        static let name = "Main"
        static let code = 4
    }
}
```